catalog-info.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
     - url: "https://docs.openedx.org"
       title: "Documentation"
       icon: "Web"
+  annotations:
+    openedx.org/release: "2u/release"    
```

Comment:
```suggestion
    openedx.org/release: "master"    
```

Reply:
I think given the issues we saw today and the fact that we have a month of testing after the release is cut, it's less important that we cut off of what 2U has released, if they do run into issues, we will hopefully backport the fixes at that point.

@salman2013 @farhaanbukhsh as release manager I think you have the final say on this, what do you think?